deploy/helm/sumologic/templates/logs/collector/otelcol/daemonset.yaml

```diff
@@ -77,7 +77,7 @@ spec:
         - mountPath: /var/lib/docker/containers
           name: varlibdockercontainers
           readOnly: true
-        - mountPath: {{ .Values.otellogs.config.extensions.file_storage.directory }}
+        - mountPath: /var/lib/storage/otc
```

Comment:
So it is now not possible to customize the path to OTC storage? We can add this option later though, right?

Reply:
I used to have a separate setting for this, but I ultimately decided it didn't add any value, and only made things more confusing in the context of the log collector, where there's no persistent queue, the volume is a hostPath, and it's only used for storing the filelog receiver state.

For metadata, I think this configuration can be moved to `sumologic.logs`, like in #2444 .

Reply:
Fair enough, it seems there is no need to modify this path 👍